src/SDKs/_metadata/powerbidedicated_resource-manager.txt

```diff
@@ -1,11 +1,20 @@
-2017-11-08 22:41:50 UTC
-
+﻿Installing AutoRest version: latest
```

Comment:
A file named SDKInfo_*.cs should be created with the rest of the SDK. Commit this file.
Also run `msbuild build.proj /t:Build /p:Scope=SDKs\PowerBIDedicated` and commit any changes generated

Reply:
SDKInfo_*.cs does not created after run .\generate.ps1 and also after the build..

Reply:
created. thanks